src/index.test.ts

```diff
@@ -1,6 +1,8 @@
+import { describe, it, expect } from 'vitest';
```

Comment:
Vitest doesn't inject globals like `describe`, `it`, `expect` by default. We could enable a setting for this, but I think importing is preferred over using globals.